v5/apps/based/menubar/menubar.css

```diff
@@ -378,4 +378,72 @@
 
   }
   
+}
+
+/* Container styling */
+.volume {
+  position: relative;
+  display: inline-block;
+  cursor: pointer;
+}
+
+/* Slider container appears below the icon */
+.volumeSliderContainer {
+  display: none;
+  position: absolute;
+  top: 120%;
+  /* below the icon */
+  left: 50%;
+  transform: translateX(-50%);
+  background: rgba(20, 20, 20, 0.95);
+  padding: 8px 12px;
+  border-radius: 6px;
+  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.4);
+  z-index: 999;
+}
+
+/* Slider width */
+#toggleVolumeSlider {
+  width: 100px;
+  margin: 0 auto;
+}
+
+#toggleVolumeSlider.ui-slider {
+  height: 6px;
+  /* thinner track */
+}
+
+/* Progress (white) */
+.ui-slider-horizontal .ui-slider-range {
+  background: #fff;
+  border-radius: 2px;
+}
+
+/* Tooltip BELOW the slider */
+.slider-value {
```

Comment:
Same idea here. We should have `.slider-value` scoped to the specific slider, it's possible somewhere else in the app we would have a `.slider-value` class.